src/styles/main.scss

```diff
@@ -1,3 +1,20 @@
 body {
   margin: 0;
 }
+
+@import "./utils/mixins";
+@import "./utils/variables";
+
+@import "./blocks/reset.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/catalog.scss";
+@import "./blocks/card.scss";
+@import "./blocks/star.scss";
+
+html {
```

Comment:
This changes apply after all your changes in block scss.
Move it before your imports